packages/backend/src/peripherals/database/migrations/107_extend_finality.ts

```diff
@@ -0,0 +1,26 @@
+/*
+                      ====== IMPORTANT NOTICE ======
+
+DO NOT EDIT OR RENAME THIS FILE
+
+This is a migration file. Once created the file should not be renamed or edited,
+because migrations are only run once on the production server.
+
+If you find that something was incorrectly set up in the `up` function you
+should create a new migration file that fixes the issue.
+
+*/
+
+import { Knex } from 'knex'
+
+export async function up(knex: Knex) {
+  await knex.schema.alterTable('finality', function (table) {
+    table.integer('average_state_update')
```

Comment:
It should be nullable

Reply:
https://github.com/l2beat/l2beat/pull/3502/commits/7967cf4c9752fbb77d209a183af1dde7a72ce05d